boa_ast/src/expression/regexp.rs

```diff
@@ -0,0 +1,94 @@
+//! This module contains the ECMAScript representation regular expressions.
+//!
+//! More information:
+//!  - [ECMAScript reference][spec]
+//!  - [MDN documentation][mdn]
+//!
+//! [spec]: https://tc39.es/ecma262/#sec-regexp-regular-expression-objects
+//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions
+
+use std::ops::ControlFlow;
+
+use boa_interner::{Interner, Sym, ToInternedString};
+
+use crate::{
+    try_break,
+    visitor::{VisitWith, Visitor, VisitorMut},
+};
+
+use super::Expression;
+
+/// Regular expressions in ECMAScript.
+///
+/// More information:
+///  - [ECMAScript reference][spec]
+///  - [MDN documentation][mdn]
+///
+/// [spec]: https://tc39.es/ecma262/#sec-regexp-regular-expression-objects
+/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions
```

Comment:
This should link to the regular expression literal syntax instead.
https://tc39.es/ecma262/#sec-literals-regular-expression-literals